setuptools/dist.py

```diff
@@ -406,7 +411,8 @@ class Distribution(_Distribution):
         'long_description_content_type': None,
         'project_urls': dict,
         'provides_extras': ordered_set.OrderedSet,
-        'license_files': ordered_set.OrderedSet,
+        'license_file': None,
+        'license_files': None,
```

Comment:
Here it probably makes sense to just declare the default here rather than declare a null value and then set a default if the value is a null value.

Reply:
That was my initial idea as well. However, with that we can't determined if the user didn't specify the option (thus the defaults would be loaded) or left it empty (in which case on license files would be included). Both would equal an empty list.

Reply:
Oh, that's very interesting. And the test cases capture this aspect. Consider me impressed.